packages/firestore/test/integration/api/bundle.test.ts

```diff
@@ -51,74 +45,133 @@ function verifyInProgress(
   expect(p.documentsLoaded).to.equal(expectedDocuments);
 }
 
-apiDescribe('Bundles', (persistence: boolean) => {
+/**
+ * Returns a testing bundle string for the given projectId.
+ *
+ * The function is commented out, but kept for documentation purpose. It accesses SDK
+ * internals, which is not available in test:minified.
+ *
+ * The tests uses `BUNDLE_TEMPLATE` as test data instead, which is generated from this function
+ * and replaced with different project IDs when required. To update `BUNDLE_TEMPALTE`, you
+ * need to uncomment the function, and copy/paste from the console output to `BUNDLE_TEMPALTE`.
+ * It is manual, but should be required only rarely.
+ */
+/*
+function bundleWithTestDocsAndQueries(projectId: string): string {
```

Comment:
Can you move this method to one of our helper/util classes and uncomment it? If you leave it here and leave it commented out, the function will break as we refactor the surrounding code.